mobile/test/objective-c/EnvoyTestServer.mm

```diff
@@ -0,0 +1,39 @@
+#import "test/objective-c/EnvoyTestServer.h"
+#import "test/common/integration/test_server_interface.h"
+
+@implementation EnvoyTestServer
+
+// Starts a server with HTTP3 and TLS.
```

Comment:
nit: Comments like this which describe what the function does belong in the header file where they are declared, not the source file where they are defined. Since you've added them to EnvoyTestServer.h you can removed them from here.

Reply:
Removed